test/client/fixtures/redirect-watch-test.js

```diff
@@ -18,31 +17,35 @@ QUnit.testDone(function () {
     settings.get().sessionId = storedSessionId;
 });
 
-function changeLocation (locationChangeScript) {
+function redirectIframe (redirectScript, stayIframe, iframeName) {
     return new Promise(function (resolve, reject) {
-        var iframe   = document.createElement('iframe');
-        var resolved = false;
+        var iframe = document.createElement('iframe');
 
-        iframe.id  = 'test' + Date.now();
-        iframe.src = location.protocol + '//' + location.host + '/unchangeableUrlSession!i/' + iframeLocation;
+        iframe.id   = 'test' + Date.now();
```

Comment:
Maybe we do need a `generateUniqueId` function in test utils?
For  example https://gist.github.com/gordonbrander/2230317